### DISCUSSION TITLE
Opacity setting for annotations

### DISCUSSION THREAD

**Discussion by oliversturm** [Softal-io/presentify-community#11] [Ideas]
I see that the cursor highlight feature has an opacity setting, but unfortunately the annotation feature does not. I would generally like annotations to be a bit more opaque than they are, as a matter of personal preference -- they would just stick out more on screen! 

However, more importantly, I've just had the experience that a projector in a conference hotel had a rather washed-out image and the low-contrast opacity of the annotations meant that they were practically invisible on the projector screen. Please add a setting so I can change opacity of annotations up to 100%!

**Comment by rampatra** [comment]
The annotations are 100% opaque. I think you have enabled Highlighter mode by mistake. Please press H to disable it. 

Press ? to see all other key shortcuts.

**Reply by oliversturm** [reply]
Fascinating! ;-)

You are correct and that fixes the immediate problem. I still think an opacity setting would be useful, but that's now a different point. 

I don't think I've ever used the app without that highlighter mode -- how would you even know it exists, since it's not in the menu or anywhere I can see? 

Thanks for the help!

**Reply by rampatra** [reply]
When you activate it, you should see a pop-up telling you that the Highlight mode is activated. But yes, I can show this info somewhere else too because if someone misses the pop-up, they might get confused later.

A side note, you can see all the features/key shortcuts by pressing `?` key when in Annotation mode.

**Comment by larryebaum** [comment]
Having controllable opacity for annotations would be nice to have... Being able to 'highlight' text and not obscure what is highlighted is useful, especially if we can get additional 'highlighter' tool with wider pen size that is opacity controllable.
